src/services/strike_service.py

```diff
@@ -120,6 +120,27 @@ def _apply_strike_point_penalty(db_user: User, severity: StrikeSeverity):
             )
 
 
+def _calculate_punishment(previous_points: int, total_points: int) -> int:
+
+    thresholds = [(10, 14), (7, 7), (5, 3), (3, 1)]
+
+    punishment_days = 0
+    speedrun = False
```

Comment:
LOL i dont really think speedrun should be the name of the variable. i think something like `jumped_multiple_thresholds`